pkg/tools/tools.go

```diff
@@ -81,6 +82,14 @@ func initMap() {
 
 	butane := butane.New()
 	toolMap[butane.Name()] = butane
+
+	gcloud, err := gcloud.New()
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Encountered error while initializing the 'gcloud' tool: %v\n", err)
+		fmt.Fprintln(os.Stderr, "Unable to install, upgrade, or remove 'gcloud' until the error is resolved")
+	} else {
+		toolMap[gcloud.Name()] = gcloud
+	}
```

Comment:
This is a break from our typical initialization, since `gcloud.New()` potentially returns an error. 

I've explicitly opted to handle the error here instead of panicking within the `gcloud.New` method, since doing so would block upgrade/installation of other, unrelated, tools due to some problem w/ the google sdk